src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstiersqstool/service/QueueAdminService.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.hmppstiersqstool.service
+
+import com.amazonaws.services.sqs.AmazonSQS
+import com.amazonaws.services.sqs.model.DeleteMessageRequest
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+
+@Service
+class QueueAdminService(
+
+  private val eventAwsSqsClient: AmazonSQS,
+  private val eventDlqAwsSqsClient: AmazonSQS,
+  @Value("\${offender-events.sqs-queue}") private val eventQueueUrl: String,
+  @Value("\${offender-events.sqs-dlq}") private val eventDlqUrl: String,
+) {
+
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  fun transferEventMessages() =
+    repeat(1) {
```

Comment:
Whats this? is it part of their DSM? repeat 1 feels redundant but I've no idea.

Reply:
It will do all of them - there's a function further down to get the message count which will be used here. I just wanted to do one to start with to check it works OK